Compiler/src/verifytrim.jl

```diff
@@ -189,7 +189,6 @@ function verify_codeinstance!(interp::NativeInterpreter, codeinst::CodeInstance,
             end
             # TODO: check for calls to Base.atexit?
         elseif isexpr(stmt, :call)
-            error = "unresolved call"
```

Comment:
This makes things less parallel structured here (normally the function sets the error message, then uses continue to skip it, instead of attempting to catch every possible else clause)